guide/designing-products/open-design.md

```diff
@@ -51,68 +52,105 @@ Illustration sources
+- Join the conversation on [Slack]({{site.slack_invite_url}}). Navigate toward the channels that interest you. Dive into topics and conversations that draw you in.
+- Identify your strengths and goals. Figure out how they may fit within the community's vision. A good place to start is listening to a [Community Call](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aissue+is%3Aopen+Community+Call) or reading our [newsletter](https://bitcoindesign.substack.com/).  
+- Feel free to ask questions or ask for help in solving bitcoin design issues.
+- See some of the community [Projects](https://bitcoin.design/projects/).
+- Visit our [Contribute](https://bitcoin.design/contribute/) page.
 
+### Test, redesign, and engage in feedback
 
-- Join the [Bitcoin Design Community]({{site.slack_invite_url}}) on Slack and see what others are discussing. A good way to get started is by asking questions and joining conversations that interest you. Our [Community Calls](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aissue+is%3Aopen+Community+Call) and [Design Review](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aissue+is%3Aopen+Design+Review+Call+) calls are also great ways to get active.
-- Download a Bitcoin wallet and experiment with small amounts of bitcoin. You can also use Testnet, which allows you to use all the features of the real Bitcoin network, but with worthless test bitcoin.
-- Redesign a bitcoin application that interests you. Even as a brief exercise, a redesign is a great way to develop a deeper understanding of how something works and why specific decisions were made.
-- Read, watch and listen. There is a lot of great material out there to learn more about Bitcoin. Unfortunately, design-focused content is not as common, but expanding into related materials (like finance) might help.
-- Review your tools and workflows and consider how you can make them more public and accessible to others.
+- Experiment with Bitcoin and Lightning. Download a wallet and test small amounts. You can also test network features without using real bitcoin on Testnet. It works with some wallets.
+- Redesign a bitcoin and/or lightning application or focus on a CX issue that interests you. Even as a brief exercise, a redesign is a great way to develop understanding of how something works.
+- Join a [Design Review](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aissue+is%3Aopen+Design+Review+Call+). A call focusing on giving feedback to improve a specific bitcoin product. Products vary by call.
+- Join the [Wallet Improvement Project](https://github.com/BitcoinDesign/Guide/issues/493). It tests a list of real world bitcoin projects and suggests redesigns to their makers.  
+
+### Continuous learning
+
+Read, watch, listen, and test. You can discover great material about Bitcoin and related topics (like finance). You may not find as much material on the design side of bitcoin applications.
```

Comment:
Would be cool if we had some starting points, like a YouTube playlist with good design-related videos, or a few podcast episodes to link to.

Reply:
Do we really have such resources could you provide a specific suggestion with the links, as I'm not really 100% sure what to add here.